### DISCUSSION TITLE
FLC Hold Repeatable?

### DISCUSSION THREAD

**Discussion by ourduck** [nguyenquyhy/Flight-Tracker-StreamDeck#376] [General]
Great product, really enjoying the A/P panel.  A couple of questions:

Can the FLC long hold be changed to be repeatable?  ALT does this but VS and FLC do not. 
Is there a place to get the icons to edit?  I'd like to add the yellow bars on the HDG button like the AP button so they all look the same when on.

Thanks!

**Comment by nguyenquyhy** [comment]
> Can the FLC long hold be changed to be repeatable? ALT does this but VS and FLC do not.

I don't really understand this. Are you talking about the +/- button?

> Is there a place to get the icons to edit?

You can find the images we are using here:
https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/tree/main/FlightStreamDeck.AddOn/Images

**Reply by ourduck** [reply]
Yes, the +/- for some of the buttons are not long press like the heading button. The VS and FLC require individual presses each increment.  As they are not customizable I can’t add the “repeat” button. 

Thank you so much, these work so well. Thank you for sharing! I enjoy your product and it’s really added to the flight sim!

**Reply by nguyenquyhy** [reply]
Ah this is a bug and is certainly not intended. VS, Airspeed suppose to work the same as Heading and Altitude. They should work properly now in latest version 0.47. 

I missed out the Vertical Speed/Airspeed function, so the fix for that one will come in the next update.

**Reply by nguyenquyhy** [reply]
Created https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/issues/377

**Comment by ourduck** [comment]
![F3E038F8-3580-46D6-BCC0-5E2F53238DFC](https://user-images.githubusercontent.com/80160370/207216592-ae6e1676-4f00-4b53-a2b5-1a23c2404395.png)
